code/__DEFINES/~~~splurt_defines/bodyparts.dm

```diff
@@ -0,0 +1,4 @@
+#undef LIMB_MAX_HP_DEFAULT
+#undef LIMB_MAX_HP_CORE
+#define LIMB_MAX_HP_DEFAULT 60 //Used by most all limbs by default. // SPLURT EDIT CHANGE - ORIGINAL : #define LIMB_MAX_HP_DEFAULT 50
```

Comment:
If it was 50 on 100 health, and 60 on 135, it should probably go only to like 56 on 120
```suggestion
#define LIMB_MAX_HP_DEFAULT 56 //Used by most all limbs by default. // SPLURT EDIT CHANGE - ORIGINAL : #define LIMB_MAX_HP_DEFAULT 50
```